### ISSUE TITLE
Remediate item image carousel, previous and next buttons, etc.

### ISSUE THREAD

**Issue by erinchambers** [CDRH/genoa_mukurtu#101]
_**Note:** This issue is moot if the documents don't get transcribed and we choose to archive the site._

There are a wide variety of WCAG 2.1 violations with the way item images are displayed/navigated through. Here's a recording of macOS VoiceOver in _read_ mode. You'll notice pretty readily that the image filenames are read aloud instead of alt text or descriptions.

https://github.com/user-attachments/assets/a79436b9-6b2e-44b6-9368-e578365b693b

What might not be readily obvious is that screen reader users aren't told how many images there are, nor are the images in the tab order of the page. If you navigate through the page with the keyboard, tabbing sends the focus straight from the Previous link to the Next link and then to the main image below, skipping the images in the carousel. There _is_ a way to access the currently visible images—navigating to either Previous or Next and then using the left and right arrow keys—but if that's how we want to do it, the carousel should have a title or `aria-label` announcing to screen readers that it is, in fact, a carousel (and that's why the arrow keys are used).

Here's a recording of VoiceOver as I tab through the page (sorry about the cutoff words! ignore those):

https://github.com/user-attachments/assets/8242bbb4-71fb-490c-acb1-160da3149340
